azure/common/pr.yml

```diff
@@ -45,9 +45,10 @@ stages:
   - ${{ if parameters.deploy_review_sandbox }}:
```

Comment:
This won't work.

You need to remove the effect of the deploy_review_sandbox variable.
The PR template is called inside a loop in deploy-stages.yml

```yaml

stages:
  - ${{ each apigee_deployment in parameters.apigee_deployments }}:
    - template: ./${{ parameters.deploy_template }}
      parameters:
        ${{ insert }}: ${{ apigee_deployment }}
        ${{ each param in parameters }}:
          ${{ if notIn(param.key, 'deploy_template', 'apigee_deployments', 'deploy_re
```
So now there's an internal-dev-sandbox environment want the PR template to just do it's thing for that environment alone, so delete the entire ```deploy_review_sandbox``` conditional.